docs/how-to-guides/how-to-create-basic-restful-api.md

```diff
@@ -16,7 +16,7 @@ everything steams from.
 // app.ts
 const app = new Application();
 
-await app.listen();
+app.listen();
```

Comment:
No top level await needed? It would be nice to give clients the assertion that the app successfully started listening on the port and everything is up and running smoothly.

Reply:
Ah I see where you're getting at! The return type of this is actually a server which gives all the assertions you'd need? It works the same as the Deno Server instance :) Happy to explain more!